ci/requirements/py36-min-all-deps.yml

```diff
@@ -40,7 +40,7 @@ dependencies:
   - pytest-cov
   - pytest-env
   - rasterio=1.0
-  - scipy=1.0  # Policy allows for 1.2, but scipy>=1.1 breaks numpy=1.14
+  - scipy=1.2  # Policy allows for 1.3
```

Comment:
Why not use 1.3 then?

Reply:
I was getting segmentation faults locally. I'll bump once I fix the tests.